fake_localization[] localization_map

float64 elapsed